packages/core/src/Footer/Initiative.js

```diff
@@ -1,33 +1,12 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-import { makeStyles } from "@material-ui/core/styles";
-
 import A from "../A";
 import RichTypography from "../RichTypography";
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    width: "100%",
-  },
-  supporterLogo: {
-    width: "9.6275rem",
-    height: "auto",
-    [theme.breakpoints.up("md")]: {
-      width: "13.7375rem",
-    },
-  },
-  support: {
-    display: "block",
-    marginBottom: "1.618125rem",
-    marginTop: "1.708rem",
-    [theme.breakpoints.up("md")]: {
-      display: "none",
-    },
-  },
-}));
+import useStyles from "./useStyles";
 
-function Initiative({ logo, initiative, ...props }) {
+function Initiative({ logo, children, ...props }) {
```

Comment:
```suggestion
function Initiative({ children, image, url, ...props }) {
```
So here we just deal with the logo props dirrectly